### PR TITLE
Explain transcription convention in docstring

### LINKED ISSUE #4438: Bio.Seq.transcribe does not evaluate to naturally explained DNA transcription
### Setup

I am reporting a problem with Biopython version 1.81, Python version 3.10.4, and operating
system as follows:

```python
sys.version =  3.10.4 (tags/v3.10.4:9d38120, Mar 23 2022, 23:13:41) [MSC v.1929 64 bit (AMD64)]
platform.python_implementation = CPython
platform.platform = Windows-10-10.0.22621-SP0
```

### Expected behaviour

Since transcription is defined as "the process by which the information in a strand of DNA is copied into a new molecule of messenger RNA by an enzyme called RNA polymerase and a number of accessory proteins called transcription factors" I would expect a DNA sequence like "ATCGATCGATCGAGATCGGATCGAT" to be transcribed as "UAGCUAGCUAGCUCUAGCCUAGCUA". 
This would mean that DNA is treated as "Template DNA" and the RNA output would reflect the real product of RNA polymerase. 

### Actual behaviour

If I run the same DNA sequence in Bio.Seq.transcribe() this will evaluate to "AUCGAUCGAUCGAGAUCGGATCGAU". This, even if it reflects the DNA strand in RNA bases it is not what "DNA transcription" biologically does

### Steps to reproduce

```python
import Bio.Seq

dna_seq = "ATCGATCGATCGAGATCGGATCGAT"
current_transcription_output = Bio.Seq.transcribe(dna_seq)  # evaluates to AUCGAUCGAUCGAGAUCGGAUCGAU
```

In  order to achieve the correct result it would required to write as follows:
```python
def transcribe(dna: str) -> str:
    rna_list = []
    dna_2_rna_base_match_table = {
        "T": "A",
        "A": "U",
        "C": "G",
        "G": "C"
    }
    for base in dna:
        rna_list.append(dna_2_rna_base_match_table.get(base, "."))
    return "".join(rna_list)


dna_seq = "ATCGATCGATCGAGATCGGATCGAT"
correct_transcription_output = transcribe(dna_seq)  # evaluates to UAGCUAGCUAGCUCUAGCCUAGCUA
```

### PR DESCRIPTION
**Pull request by peterjc**
This reuses a paragraph from our tutorial.

Closes #4438

- [x] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [x] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit``
locally, and understand that continuous integration checks will be used to
confirm the Biopython unit tests and style checks pass with these changes.

- [x] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Review by mdehoon — COMMENTED** [review]
How about

"Following the usual convention, the sequence is interpreted as the coding strand, not the template strand), of the DNA double helix."

File: Bio/Seq.py [hunk 1]
```diff
@@ -2821,11 +2819,9 @@ def defined_ranges(self):
...
     """Transcribe a DNA sequence into RNA.
 
-    The actual biological transcription process works from the template
-    strand, doing a reverse complement (TCAG to CUGA) to give the mRNA.
-    However, in Biopython and bioinformatics in general, we typically work
-    directly with the coding strand because this means we can get the mRNA
-    sequence just by switching T to U.
+    Following the usual convention, the sequence is interpreted as the
+    coding strand of the DNA double helix, not the template strand. This
+    means we can get the mRNA sequence just by switching T to U.
```
Comment (mdehoon) [inline Bio/Seq.py:2824]: It's not necessarily an mRNA.

Comment (peterjc) [reply Bio/Seq.py:2824]: So just drop the "m" then?

Comment (mdehoon) [reply Bio/Seq.py:2824]: Yes that is fine.

**Pull request merged**